core/chains/evm/gas/fixed_price_estimator.go

```diff
@@ -11,7 +11,7 @@ import (
 	"github.com/smartcontractkit/chainlink/v2/core/logger"
 )
 
-var _ EvmEstimator = &fixedPriceEstimator{}
+var _ EvmEstimator = (*fixedPriceEstimator)(nil)
```

Comment:
refactoring this to follow updated standard which saves on memory? 

Still not 100% sure why this is the new approach

Reply:
It should be compiled away regardless, but this format is more direct because the point is to assert the type (`*fixedPriceEstimator`), and taking the address via `&fixedPriceEstimator` is just an indirect way to get at that type.

Reply:
thank you for the explaination :)